files/default/templates/hopsworks.email.tmpl

```diff
@@ -0,0 +1,420 @@
+}
+
+/* -------------------------------------
+    RESPONSIVE AND MOBILE FRIENDLY STYLES
+------------------------------------- */
+@media only screen and (max-width: 640px) {
+  body {
+    padding: 0 !important;
+  }
+
+  h1, h2, h3, h4 {
+    font-weight: 800 !important;
+    margin: 20px 0 5px !important;
+  }
+
+  h1 {
+    font-size: 22px !important;
+  }
+
+  h2 {
+    font-size: 18px !important;
+  }
+
+  h3 {
+    font-size: 16px !important;
+  }
+
+  .container {
+    padding: 0 !important;
+    width: 100% !important;
+  }
+
+  .content {
+    padding: 0 !important;
+  }
+
+  .content-wrap {
+    padding: 10px !important;
+  }
+
+  .invoice {
+    width: 100% !important;
+  }
+}
+</style>
+</head>
+<body itemscope itemtype="http://schema.org/EmailMessage">
+<table class="body-wrap">
+  <tr>
+    <td></td>
+    <td class="container" width="600">
+      <div class="content">
+        <table class="main" width="100%" cellpadding="0" cellspacing="0">
+          <tr>
+            {{ if gt (len .Alerts.Firing) 0 }}
+            <td class="alert alert-warning" style="background-color: #E6522C;">
+            {{ else }}
+            <td class="alert alert-good" style="background-color: #68B90F;">
+            {{ end }}
+              {{ .Alerts | len }} alert{{ if gt (len .Alerts) 1 }}s{{ end }} for {{ range .GroupLabels.SortedPairs }}
+                {{ .Name }}={{ .Value }} 
+              {{ end }}
+            </td>
+          </tr>
+          <tr>
+            <td class="content-wrap">
+              <table width="100%" cellpadding="0" cellspacing="0">
+                {{ if gt (len .Alerts.Firing) 0 }}
+                <tr>
+                  <td class="content-block">
+                    <strong>[{{ .Alerts.Firing | len }}] Firing {{ .CommonLabels.alertname }}</strong>
+                  </td>
+                </tr>
+                {{ end }}
+                {{ range .Alerts.Firing }}
+                <tr>
+                  <td class="content-block">
+                    {{ if gt (len .Annotations) 0 }}
+                      <strong>Alert:</strong> {{ .Annotations.title }} {{ if .Labels.severity }} - {{ .Labels.severity }} {{ end }}<br />
+                      <strong>Summary:</strong> {{ .Annotations.summary }} <br />
+                      <strong>Description:</strong> {{ .Annotations.description }} <br />
+                    {{ end }}
+                    <strong>Details:</strong><br />
+                    <ul>
+                      <li> Project: {{ .Labels.project }} </li>
+                      {{ if .Labels.job -}} <li> Job: {{ .Labels.job }} </li> {{ end }}
+                      {{ if .Labels.executionId -}} <li>  Execution id: {{ .Labels.executionId }} </li> {{ end }}
+                      {{ if .Labels.featureGroup -}} <li>  Feature Group: {{ .Labels.featureGroup }} </li> {{ end }}
+                      {{ if .Labels.featureGroupId -}} <li>  Feature Group id: {{ .Labels.featureGroupId }} </li> {{ end }}
```

Comment:
Feature group id is useless. The alert should contain feature store name/ feature group name/feature group version.